docs/textinput.md

```diff
@@ -513,6 +513,8 @@ Callback that is called when the text input's submit button is pressed with the
 | -------- | -------- |
 | function | No       |
 
+Note that on iOS this method was never called when use `keyboardType="phone-pad"`.
```

Comment:
Could you change this to:

```
Note that on iOS this method is not called when using `keyboardType="phone-pad"`.
```

Reply:
Sure
https://github.com/facebook/react-native-website/pull/1527